### PR TITLE
Add accountNumber and sortCode length validation

### PR DESCRIPTION
**Pull request by promag**
This PR adds `accountNumber` and `sortCode` length validation after sanitisation.

The `sortCode` must have 6 digits. The `accountNumber` must have 6 to 10 digits.

### CODE REVIEW

File: test/index_test.js [hunk 1]
```diff
@@ -7,6 +7,10 @@ import UkModulusChecking from './../src';
 
 const accounts = {
   invalid: [
+    { accountNumber: '12345', sortCode: '123456' },
+    { accountNumber: '12345789012', sortCode: '123456' },
+    { accountNumber: '12345789', sortCode: '12345' },
+    { accountNumber: '15763217', sortCode: '938063' },
```
Comment (franciscocardoso) [inline test/index_test.js]: Duplicated. Please remove.

File: test/index_test.js [hunk 2]
```diff
@@ -7,6 +7,10 @@ import UkModulusChecking from './../src';
 
 const accounts = {
   invalid: [
+    { accountNumber: '12345', sortCode: '123456' },
```
Comment (franciscocardoso) [inline test/index_test.js]: Can we find some examples of valid account numbers to add to tests?

File: src/index.js [hunk 3]
```diff
@@ -199,6 +199,14 @@ export default class UkModulusChecking {
    */
 
   isValid() {
+    if (this.sortCode.length !== 6) {
+      return false;
+    }
+
+    if (this.accountNumber.length < 6 || this.accountNumber.length > 10) {
```
Comment (franciscocardoso) [inline src/index.js]: WDYT about test `accountNumber` first? Or:

```js
if (this.accountNumber.length < 6 || this.accountNumber.length > 10 || this.sortCode.length !== 6) {
  return false;
}
```

Comment (promag) [inline src/index.js]: What about `if(!_.inRange(this.accountNumber, 6, 11) || this.sortCode.length !== 6)`?

Comment (franciscocardoso) [inline src/index.js]: ✨ Just a quick fix:

```js
... _.inRange(this.accountNumber.length, 6, 11) ...
```

File: test/index_test.js (hunk 2 above)
Comment (promag) [inline test/index_test.js]: Can't find.

Comment (nunofgs) [inline test/index_test.js]: After discussing offline with @franciscocardoso, we'd prefer to have these special cases as actual tests.

**Pull request merged**